nibabel/tests/test_orientations.py

```diff
@@ -203,6 +203,23 @@ def test_io_orientation():
     assert_array_equal(io_orientation(def_aff, tol=0), pass_tol)
     def_aff[1, 1] = eps * 10
     assert_array_equal(io_orientation(def_aff, tol=1e-5), fail_tol)
+    # Test drop of rows, columns
+    mat, vec = to_matvec(def_aff)
+    aff_extra_col = np.zeros((4, 5))
+    aff_extra_col[:3, :3] = mat
+    aff_extra_col[:3, -1] = vec
```

Comment:
In both cases I was aiming for a 4th column / row with all zero, with the last column always being the translations, and non-zero.  Is that what you thought?